app/javascript/controllers/scrollable_controller.js

```diff
@@ -37,7 +36,7 @@ export default class extends Controller {
   }
 
   scrolled() {
-    this.debouncedCheckScroll()
+    this.throttledCheckScroll()
```

Comment:
This was causing the user-initiated scroll to be delayed and it was never catching the final end-of-scroll event, because debounce throws that one away.